dashboard/app/models/script_level.rb

```diff
@@ -21,10 +21,13 @@
 #  index_script_levels_on_stage_id   (stage_id)
 #
 
+require 'cdo/shared_constants'
+
 # Joins a Script to a Level
 # A Script has one or more Levels, and a Level can belong to one or more Scripts
 class ScriptLevel < ActiveRecord::Base
   include LevelsHelper
+  include SharedConstants
```

Comment:
:( That doesn't work either for class methods. Here's an [example](http://stackoverflow.com/a/10693010) from stackoverflow that should work.